BibBuddy/app/src/main/java/de/bibbuddy/NoteRecyclerViewAdapter.java

```diff
@@ -24,18 +34,34 @@
     extends RecyclerView.Adapter<NoteRecyclerViewAdapter.NotesViewHolder> {
 
   private final MainActivity activity;
+  private final NoteModel noteModel;
+  private final ArrayList<MediaPlayer> mediaPlayers;
+  private final ArrayList<ImageButton> playButtons;
+  private final ArrayList<ImageButton> stopButtons;
+  private final ArrayList<ProgressBar> progressBars;
+  private final ArrayList<SeekBarListener> seekBarListeners;
   private List<NoteItem> noteList;
   private ViewGroup parent;
+  private int mediaPlayerPosition;
+  private boolean paused;
```

Comment:
Maybe some grouping?

```suggestion
  private final MainActivity activity;
  private final NoteModel noteModel;
  
  private final ArrayList<MediaPlayer> mediaPlayers;
  private final ArrayList<ImageButton> playButtons;
  private final ArrayList<ImageButton> stopButtons;
  private final ArrayList<ProgressBar> progressBars;
  private final ArrayList<SeekBarListener> seekBarListeners;
  
  private List<NoteItem> noteList;
  private ViewGroup parent;
  private int mediaPlayerPosition;
  private boolean paused;
```